css/properties/text-anchor.json

```
@@ -25,7 +27,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

```suggestion
              "version_added": "4"
```

See: https://github.com/caugner/css-properties/commit/cf1b5de97bcb5f335533005f5fdd8ff327bcfc11#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R181